srcpkgs/reddio/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'reddio'
+pkgname=reddio
+version=0.3
+revision=5
+wrksrc="reddio-v0.3"
+depends="curl jq"
+short_desc="Command-line interface for Reddit written in POSIX"
+maintainer="itamiko <itamiko@pm.me>"
+license="MIT"
+homepage="https://gitlab.com/aaronNG/reddio"
+distfiles="https://gitlab.com/aaronNG/reddio/-/archive/v${version}/reddio-v${version}.tar.gz"
+checksum=be200dfba2bbbbff8ba2445ad9f41842419c9e3d65aa103a8334886f2b1ff870
+patch_args="-Np1"
```

Comment:
you could use the default p0 if you remove the leading `a/` and `b/` from the paths in the patch